website/docs/advanced/search_path.md

```diff
@@ -3,15 +3,145 @@ id: search_path
 title: Config Search Path
 ---
 
-The Config Search Path is a list of paths that are searched in order to find configs. It is similar to
-the Python PYTHONPATH.
+import {ExampleGithubLink} from "@site/src/components/GithubLink"
 
+The Config Search Path is a list of paths that Hydra searches in order to find configs. It is similar to
+the Python `PYTHONPATH`. 
  - When a config is requested, The first matching config in the search path is used.
- - Each search path element has a schema prefix such as file:// or pkg:// that is corresponding to a ConfigSourcePlugin.
- - `SearchPathPlugin` can manipulate the search path.
+ - Each search path element has a schema prefix such as `file://` or `pkg://` that is corresponding to a `ConfigSourcePlugin`.
```

Comment:
I think we need to explain each of the prefixes.

- file:// is a file system path. it should normally be absolute, if its relative I _think_ it's relative to cwd. I think separator is forward slash (even on Windows).
- pkg:// is a python module. it should be importable and should contain and `__init__.py`. separator is a `.`.

Double check those statements and update the docs with the info.